### PR TITLE
Installation Instructions

### PR DESCRIPTION
**Pull request by dylanesque**
First pass at project installation instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,22 @@
 # police-data-trust
+# Installation
+
+1. Make sure that you have [Python 3.8](https://www.python.org/), [Git](https://git-scm.com/), and [PostgreSQL](https://www.postgresql.org/) are installed on your machine.
+
+2. Clone this repository `git clone git@github.com:codeforboston/police-data-trust.git`
```
Comment (dwreeves) [inline README.md:6]: Fork first.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,26 @@
 # police-data-trust
+# Installation
+
+1. Make sure that you have [Python 3.8](https://www.python.org/), [Git](https://git-scm.com/), and [PostgreSQL](https://www.postgresql.org/) are installed on your machine.
```
Comment (dwreeves) [inline README.md:4]: "Make sure that ... are installed" or alternatively "Make sure that you have ... installed"

File: README.md [hunk 3]
```diff
@@ -1,3 +1,26 @@
 # police-data-trust
+# Installation
```
Comment (dwreeves) [inline README.md:2]: \<h2\> I think for everything that's not the project name, so `##`

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -15,9 +16,19 @@ to which all contributors must adhere.
...
-## [Pull Requests]()
+## If You're New to Open Source
+
+If you're brand new to Git and/or GitHub, you'll want to familiarize yourself with the basics of version control, which you can do with Github's [Hello World tutorial](https://guides.github.com/activities/hello-world/) 
+
+## [Making Pull Requests]()
+
+Pull requests are merged into our main branch. This makes main our main development branch. You'll want to keep your fork and local repo current with this branch so you're always working off the latest code. To do so, run these commands:
+
+git checkout main
```
Comment (dwreeves) [inline CONTRIBUTING.md:27]: Triple backtick:

\`\`\`
 git checkout main
 git pull upstream main
 git push origin main
\`\`\`

File: README.md (hunk 1 above)
Comment (dylanesque) [reply README.md]: Making this change, and all previously suggested changes in the PR now.

**Review by DMalone87 — APPROVED** [review]

**Review by DMalone87 — APPROVED** [review]

**Pull request merged**